src/schema/v2/partners.ts

```diff
@@ -139,17 +139,32 @@ export const Partners: GraphQLFieldConfig<void, ResolverContext> = {
 export const PartnersConnection: GraphQLFieldConfig<void, ResolverContext> = {
   type: connectionWithCursorInfo({ nodeType: PartnerType }).connectionType,
   description: "A list of Partners",
-  args: pageable({
-    ids: {
-      type: new GraphQLList(GraphQLString),
-    },
-  }),
+  args: pageable(
+    Object.assign(
```

Comment:
#minor `Object.assign` may not be needed, might be able to use spread (...) instead.

``` ts
pageable(
  {
    ids: .....,
    ...pick(.....),
  }
```

Reply:
Fixed